paths/articles/comments.yaml

```diff
@@ -1,12 +1,10 @@
 x-swagger-router-controller: mock
 get:
-  description: >
+  description: |
```

Comment:
this preserves the new lines, is that intended? 
https://stackoverflow.com/a/21699210

Reply:
Didn't really consider how we should display the descriptions, but after checking I agree it shouldn't have new lines